test/skimNtuple2017_HHbtag.cpp

```diff
@@ -119,60 +119,20 @@ const float DYscale_MTT_HighPt_err [3] = {0.0030012172, 0.046576904 , 0.03594971
 const float DYscale_MTT_vHighPt_err[3] = {0.0061768066, 0.15213682  , 0.091582069 };
 
 /* NOTE ON THE COMPUTATION OF STITCH WEIGHTS:
+** 'Old' approach:
 ** - to be updated at each production, using the number of processed events N_inclusive and N_njets for each sample
 ** - say f_i is the fraction of inclusive events in the i bin on njets (can be 2D nb-njet as well)
 ** = then sigma_i = f_i * sigmal_LO
 ** - stitchWeight (0jet) = f_0 / (f_0 * N_inclusive)
 ** - stitchWeight (njet) = f_n / (f_n * N_inclusive + N_njets)
+**==============================================================
+** new approach for NLO:
+** 3 samples merged: Inclusive, Njet-sliced, pT-sliced.
+** merging done according to cross-section from xsdb
+** [0,50] GeV pT slice is buggy and not included in the stitching
+** -> 2 dedicated scale factors are needed (for pT>50 and pT<50) to restore the total cross-section
```

Comment:
The bug in the [0,50] GeV sample was that the cut was > 0 GeV instead of >= 0 GeV, correct? So couldn't we use that sample as well and only use the factor 1/2 for the 0 GeV bin?

Also regarding this new approach we were wondering about the jet binned samples. The inclusive and the LHE Z pt binned samples cover the whole phase space, but the jet binned ones only 0,1 and 2 jets. Is this not a problem with this approach as from a certain number of jets the phase space is only covered by 2 samples again and not 3, correct?

Reply:
Ah okay, yes that makes sense. Thanks